### ISSUE TITLE
Installation problem

### ISSUE THREAD

**Issue by karanshgl** [phage-nz/ph0neutria#10]
I am having issues with installation. When I try to install,  get this:
usermod: user 'clamav' does not exist
sed: can't read /etc/clamav/clamd.conf: No such file or directory
./install.sh: line 62: /etc/init.d/clamav-daemon: No such file or directory

**Comment by karanshgl** [comment]
run.py closes automatically after showing util.VirusTotal[25397] CRITICAL Failed to query VirusTotal.
I have added the API key for VirusTotal.

**Comment by phage-nz** [comment]
It doesn't look like your clamav install completed successfully, as suggested by the lack of a clamav user, config file and daemon. What OS and OS version are you running? Are the clamav and clamav-daemon packages even available via apt for you? Did you observe any errors after the portion of the install that installs clamav? As clamav is used as a module in Viper, it's not necessary for the operation of either ph0neutria or Viper.

To be able to advise on the reason for the VirusTotal failure I'll need to see the full log output of the failure. It should also return a failure from VirusTotal in the response body:
```
logging.critical('Failed to query VirusTotal.')
logging.warning(response.text)
sys.exit(1)
```
That should make things pretty clear. Chances are you blew your request threshold.

**Comment by phage-nz** [comment]
Think I worked out what may have caused the clamav issue. You should have seen some errors from apt concerning the 'pcre' package - which is not named as such in Ubuntu - this will have caused the installation of clamav not to take place. Addressed and re-tested in Ubuntu 16.04. Try the latest version of install.sh: https://raw.githubusercontent.com/phage-nz/ph0neutria/master/install.sh
